internal/services/users-service.go

```diff
@@ -56,6 +57,22 @@ func (s *UsersService) NewUser(input ports.CreateUserInput) *customErrors.Intern
 	return nil
 }
 
+func (s *UsersService) GetUser(id string) (*entities.User, *customErrors.InternalError) {
```

Comment:
Como é uma busca por id, é bom especificar no nome do método. Pois no futuro poderemos ter outras buscas, apesar de improvável.
```suggestion
func (s *UsersService) GetUserById(id string) (*entities.User, *customErrors.InternalError) {
```